nipreps/synthstrip/wrappers/pydra.py

```diff
@@ -0,0 +1,116 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+#
+# Copyright 2022 The NiPreps Developers <nipreps@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# We support and encourage derived works from this project, please read
+# about our expectations at
+#
+#     https://www.nipreps.org/community/licensing/
+#
+"""SynthStrip interface."""
+
+import os
+import attr
+from pathlib import Path
+import pydra
+import nest_asyncio
+nest_asyncio.apply()
+
+_fs_home = os.getenv("FREESURFER_HOME", None)
+_default_model_path = Path(_fs_home) / "models" / "synthstrip.1.pt" if _fs_home else None
+
+if _fs_home and not _default_model_path.exists():
+    _default_model_path = None
+
+_SynthStripInputSpec = pydra.specs.SpecInfo(
```

Comment:
Since this is what'll be imported to be used in pydra, I think we should make it "public"

```suggestion
SynthStripInputSpec = pydra.specs.SpecInfo(
```